go/teams/loader_test.go

```diff
@@ -874,3 +874,68 @@ func TestLoaderCORE_7201(t *testing.T) {
 	})
 	require.NoError(t, err)
 }
+
+// TestLoaderCORE_8445 tests a case that came up.
+// A user had trouble loading A.B because the cached object was stuck as without RKMs.
+// U1 is an   ADMIN in A
+// U1 is only IMP implicitly in A.B
+// U1 is loads A.B caching the secret but not the RKMs
+// U1 is a    WRITER A.B
+func TestLoaderCORE_8445(t *testing.T) {
+	fus, tcs, cleanup := setupNTests(t, 2)
+	defer cleanup()
+
+	t.Logf("U0 creates A")
+	rootName, _ := createTeam2(*tcs[0])
+
+	t.Logf("U0 adds U1 to A")
+	_, err := AddMember(context.TODO(), tcs[0].G, rootName.String(), fus[1].Username, keybase1.TeamRole_ADMIN)
+	require.NoError(t, err, "add member")
+
+	t.Logf("U0 creates A.B")
+	subBName, subBID := createSubteam(tcs[0], rootName, "bbb")
+
+	t.Logf("U1 loads and caches A.B")
+	_, err = Load(context.TODO(), tcs[1].G, keybase1.LoadTeamArg{
+		ID:          subBID,
+		ForceRepoll: true,
+	})
+	require.NoError(t, err)
+
+	t.Logf("U0 adds U1 to A.B")
+	_, err = AddMember(context.TODO(), tcs[0].G, subBName.String(), fus[1].Username, keybase1.TeamRole_WRITER)
+	require.NoError(t, err, "add member")
+	t.Logf("setup complete")
```

Comment:
If there were two versions of this test, one where the order of 
`U0 adds U1 to A.B`
`U1 loads A.B without refreshing`
are swapped, would that catch some of the bugs we're discussing in loader.go?

Reply:
hmm i don't think so since when we still force `repoll=true` and get the new chain link

Reply:
i'm not sure we can force the state where we have all of the links but no rkms.. but still think it's best to have  `fetchLinksAndOrSecrets` set correctly